NEWS.md

```diff
@@ -23,6 +23,8 @@ OpenSSL 3.1
 
   * Subject or issuer names in X.509 objects are now displayed as UTF-8 strings
     by default.
+  * TCP Fast Open support is available on Windows, Linux, macOS, and FreeBSD
+    where enabled and supported.
```

Comment:
INSTALL.md says "Linux, macOS and FreeBSD" and here we say "Windows, Linux, macOS, and FreeBSD"